core/templates/dev/head/css/oppia.css

```diff
@@ -1519,12 +1513,28 @@ pre.oppia-pre-wrapped-text {
 .oppia-rte-editor > p:last-child {
   margin-bottom: 0;
 }
-.form-control.oppia-rte-content > div > p:first-child {
+
+/* The four rules below should be in sync. */
+.rte-viewer > p:first-child {
   margin-top: 0;
 }
-.form-control.oppia-rte-content > div > p:last-child {
+.rte-viewer > p:last-child {
   margin-bottom: 0;
 }
+.form-control.oppia-rte-content > div > p:first-child {
+  margin-top: 0px;
```

Comment:
Can we make this `0` this way it will look more in sync with others and consistent? (ditto below)

Reply:
Done, PTAL!